docs/assets/0123-import-scripts/README.md

```diff
@@ -0,0 +1,142 @@
+# Welcome to 0123-import-scripts!
+
+This is a template repository that can be used for the archiving process of a big dataset at the end of a research 
+project's lifetime. Download and unpack this repository from the 
+[excel2xml documentation page](https://docs.dasch.swiss/latest/DSP-TOOLS/dsp-tools-excel2xml/).  
+
+In this README, you will learn how to write a Python script for preparing data for an import into DSP.  
+
+Featuring: 
+
+- first steps with Visual Studio Code
+- the module `excel2xml` of dsp-tools
+- the benefits of Version Control with Git
+- the benefits of the Debugging Mode
+- extras: OpenRefine, Git GUIs, regexr
+
+
+# First steps with Visual Studio Code (VSC)
+[**Visual Studio Code**](https://code.visualstudio.com/download) is the industry standard IDE, free to use, and 
+recommended for your daily work at DaSCH. Recommended extensions to install from within VSC: 
+
+- redhat.vscode-xml
+- ms-python.python
+- ms-python.vscode-pylance 
+- zainchen.json
+- nickdemayo.vscode-json-editor
+- visualstudioexptteam.vscodeintellicode
+- redhat.vscode-xml
+- dotjoshjohnson.xml
```

Comment:
some thoughts on this section:
1. using the .vscode workspace settings, you can explicitly define recommended extensions, so it even prompts the user to install those extensions
2. isn't pylance by now bundled with the python extension?
3. are the two json extensions really needed for regular users?
4. is the second xml extension really needed for regular users?

I'm personally a big fan of good VS Code extensions, but if the aim is to keep it simple for non-power-users, then more extensions may be rather intimidating than helpful. But I'm not sure what the target audience of this repo is...

Reply:
Thanks for the feedback, you're right. I added a `.vscode/extensions.json` with the necessary extensions, but without pylance, and with only one json and one xml extension.